meshtastic/interfaces/ble/lifecycle_service.py

```
@@ -0,0 +1,106 @@
+from meshtastic.interfaces.ble.constants import logger
+
+if TYPE_CHECKING:
+    from meshtastic.interfaces.ble.client import BLEClient
+    from meshtastic.interfaces.ble.interface import BLEInterface
+
+
+class BLELifecycleService:
+    """Service helpers for BLEInterface lifecycle responsibilities."""
+
+    @staticmethod
+    def set_receive_wanted(iface: "BLEInterface", want_receive: bool) -> None:
```

Ruff (0.15.5)

[warning] Boolean-typed positional argument in function definition

(FBT001)